packages/cli/create-strapi-starter/utils/child-process.js

```diff
@@ -2,25 +2,27 @@
 
 const { execSync } = require('child_process');
 const execa = require('execa');
-const hasYarn = require('./has-yarn');
 const logger = require('./logger');
 
 /**
- * @param  {string} path Path to directory (frontend, backend)
+ * @param {string} path Path to directory (frontend, backend)
+ * @param {Object} options
+ * @param {boolean} options.useYarn Use yarn instead of npm
  */
-function runInstall(path) {
-  if (hasYarn()) {
-    return execa('yarn', ['install'], {
-      cwd: path,
-      stdin: 'ignore',
-    });
-  }
-
-  return execa('npm', ['install'], { cwd: path, stdin: 'ignore' });
+function runInstall(path, { useYarn }) {
```

Comment:
To make sure it's optional

```suggestion
function runInstall(path, { useYarn } = {}) {
```